test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -49,6 +50,13 @@ public void invokeFailureHandler()
                 .build();
         bootstrapAllServers(l);
 
+        // Reduce test execution time from 15+ seconds to about 8 seconds:
+        // Set aggressive timeouts for surviving MS that polls the dead MS.
+        ManagementServer ms = getManagementServer(9001);
+        ms.getCorfuRuntime().getRouter("test:9000").setTimeoutConnect(50L);
```

Comment:
1) Use the parameters. It lets you choose between 4 settings. Pick one of them - unless you can explain why TIMEOUT_LONG is inappropriate here.

 It also better be `final`, since it better not change during test execution (why would a constant ever be non-`final`?)